### PR TITLE
Filipino Translation - 1113 translated words

### PR DESCRIPTION
**Pull request by smervs**
Translating this following two files :
bake/development.rst = 814 translated words
bake/usage.rst = 173 translated words
console-and-shells/cache.rst = 25 translated words
console-and-shells/cron-jobs.rst = 101 translated words

Note : Code text, Filenames, Filepaths and computer generated text are not counted as translated words.

### CODE REVIEW

File: tl/bake/development.rst [hunk 1]
```diff
@@ -33,13 +28,11 @@ be used::
...
-If you want to modify bake from within another plugin, putting your plugin's
-bake events in the plugin ``config/bootstrap.php`` file is a good idea.
+Kung nais mong baguhin ang bake mula sa loob ng isa pang plugin, ang paglagay ng mga kaganapan ng bake sa iyong plugin na ``config/bootstrap.php`` na file ay isang magandang ideya.
 
-Bake events can be handy for making small changes to existing templates.
-For example, to change the variable names used when baking controller/template
-files one can use a function listening for ``Bake.beforeRender`` to modify the
-variables used in the bake templates::
+Ang mga kaganapan ng Bake ay maaaring madaling gamitin para sa paggawa ng maliliit na mga pagbabago sa umiiral na mga template.
+Halimbawa, upang baguhin ang mga pangalan ng variable na ginagamit habang nagbi-bake ng controller/template
```
Comment (mind-blast) [inline tl/bake/development.rst]: nagbi-bake to nag-bake

File: tl/bake/development.rst [hunk 2]
```diff
@@ -99,27 +92,27 @@ you can use the following event::
         }
     );
 
-By scoping event listeners to specific bake templates, you can simplify your
-bake related event logic and provide callbacks that are easier to test.
+Sa pamamagitan ng pagscope ng mga tagapakinig ng kaganapan sa tiyak na mga template ng bake, maaari mong gawing simple ang iyong
+lohika ng kaganapan na kaugnay ng at magbigay ng mga callbacks na mas madaling subukan.
```
Comment (mind-blast) [inline tl/bake/development.rst]: ng mga callbacks to ng mga callback

File: tl/bake/development.rst [hunk 3]
```diff
@@ -99,27 +92,27 @@ you can use the following event::
...
-to modify bake template files, is to bake a class and compare the template used
-with the pre-processed template file which is left in the application's
-**tmp/bake** folder.
+Isang paraan upang makita/maintindihan kung paano gumagana ang mga template ng bake, lalo na kapag sinusubukang  
+baguhin ang mga file na template ng bake, ay magbake na klase at ihambing ang template na ginamit 
+na may file ng pre-processed na template na naiwan sa aplikasyon na 
+**tmp/bake** na folder.
 
-So, for example, when baking a shell like so:
+Kaya, halimbawa, kapag nagbi-bake ng isang shell tulad nito:
```
Comment (mind-blast) [inline tl/bake/development.rst]: nagbi-bake to nag-bake

File: tl/bake/development.rst [hunk 4]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
 
     }
 
-.. note::
+.. tandaan::
 
-    Prior to version 1.5.0 bake used a custom erb-style tags inside .ctp template files.
+    Bago ang bersyon na 1.5.0 ang bake ay gumagamit na isang custom na er-style na mga tag sa loob ng template ng .ctp na mga file.
```
Comment (mind-blast) [inline tl/bake/development.rst]: custom to pasadya

File: tl/bake/development.rst [hunk 5]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
 .. _creating-a-bake-theme:
 
-Creating a Bake Theme
+Ang Paglikha ng isang Tema ng Bake
 =====================
 
-If you wish to modify the output produced by the "bake" command, you can
-create your own bake 'theme' which allows you to replace some or all of the
-templates that bake uses. The best way to do this is:
+Kung nais mong baguhin ang output na ginawa ng command ng "bake", maaari kang
```
Comment (robin-ho) [inline tl/bake/development.rst]: theme to tema

File: tl/bake/development.rst [hunk 6]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
+lumikha ng iyong sariling 'theme' ng bake na nagpapahintulot sa iyo na palitan ang ilan o lahat ng
```
Comment (mind-blast) [inline tl/bake/development.rst]: theme to tema

File: tl/bake/development.rst [hunk 7]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
-templates that bake uses. The best way to do this is:
+Kung nais mong baguhin ang output na ginawa ng command ng "bake", maaari kang
+lumikha ng iyong sariling 'theme' ng bake na nagpapahintulot sa iyo na palitan ang ilan o lahat ng 
+mga template na ginagamit ng bake. Ang pinakamahusay na paraan upang gawin ito ay:
 
-#. Bake a new plugin. The name of the plugin is the bake 'theme' name
-#. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/Template** to matching files in your
+#. Magbake ng isang bagong plugin. Ang pangalan ng plugin ay ang pangalan ng 'theme' ng bake
```
Comment (mind-blast) [inline tl/bake/development.rst]: theme to tema

File: tl/bake/development.rst [hunk 8]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
+#. Maglikha ng isang bagong directory **plugins/[name]/src/Template/Bake/Template/**.
+#. Kopyahin ang anumang mga template na gusto mong i-override mula sa
+   **vendor/cakephp/bake/src/Template/Bake/Template** sa pagtutugma ng mga file sa iyong
    plugin.
-#. When running bake use the ``--theme`` option to specify the bake-theme you
-   want to use. To avoid having to specify this option in each call, you can also
-   set your custom theme to be used as default theme::
+#. Kapag nagpapatakbo ng bake gamitin ang ``--theme`` na opsyon upang tiyakin ang bake-theme 
+   na gusto mong gamitin. Upang maiwasan ang pagkakaroon ng pagtiyak ng opsyon na ito sa bawat tawag, maaari mo ring
+   i-set ang iyong custom na tema na gagamitin bilang isang default na tema::
```
Comment (mind-blast) [inline tl/bake/development.rst]: i-set to itakda

Comment (mind-blast) [inline tl/bake/development.rst]: custom to pasadya

File: tl/bake/development.rst [hunk 9]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
         Configure::write('Bake.theme', 'MyTheme');
 
-Customizing the Bake Templates
+Ang Pagcustomize ng mga Template ng Bake
 ==============================
 
-If you wish to modify the default output produced by the "bake" command, you can
-create your own bake templates in your application. This way does not use the
-``--theme`` option in the command line when baking. The best way to do this is:
+Kung nais mong baguhin ang default na output na ginawa ng command ng "bake", maaari kang
```
Comment (mind-blast) [inline tl/bake/development.rst]: command  to utos

File: tl/bake/development.rst [hunk 10]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
+lumikha ng iyong sariling mga template ng bkae sa iyong aplikasyon. Sa ganitong paraan ay hindi ginagamit ang 
+``--theme`` na opsyon sa command line habang nagbi-bake. Ang pinakamahusay na paraan upang gawin ito ay:
```
Comment (mind-blast) [inline tl/bake/development.rst]: nagbi-bake to nag-bake

File: tl/bake/development.rst [hunk 11]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
 
-#. Create a new directory **/src/Template/Bake/**.
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/** to matching files in your
-   application.
+#. Maglikha ng bagong directory **/src/Template/Bake/**.
+#. Kopyahin ang anumang mga template ang gusto mong i-override mula sa
```
Comment (mind-blast) [inline tl/bake/development.rst]: i-override to sapawan

File: tl/bake/development.rst [hunk 12]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
 
-Creating New Bake Command Options
+Maglikha ng Bagong Command ng Bake na mga Opsyon
 =================================
 
-It's possible to add new bake command options, or override the ones provided by
-CakePHP by creating tasks in your application or plugins. By extending
-``Bake\Shell\Task\BakeTask``, bake will find your new task and include it as
-part of bake.
+Posibleng magdagdag ng bagong command ng bake na mga opsyon, o i-override ang mga binigay ng
```
Comment (mind-blast) [inline tl/bake/development.rst]: command to utos

Comment (mind-blast) [reply tl/bake/development.rst]: i-override to sapawan

File: tl/bake/development.rst [hunk 13]
```diff
@@ -267,15 +260,15 @@ add the following content::
...
+Ito ay bubuo ng isang bagong klase ng ``ExampleFoo`` sa **src/Foo/ExampleFoo.php**
+para gamitin sa iyong aplikasyon.
 
-If you want the ``bake`` call to also create a test file for your
-``ExampleFoo`` class, you need to overwrite the ``bakeTest()`` method in the
-``FooTask`` class to register the class suffix and namespace for your custom
-command name::
+Kung nais mo na ang ``bake`` na tawag ay maglikha din ng isang test na file para sa iyong 
+klase ng ``ExampleFoo``, kailangan mong i-overwrite ang ``bakeTest()`` na paraan sa 
+klase ng ``FooTask`` upang irehistro ang suffix ng klase at namespace para sa iyong custom
```
Comment (mind-blast) [inline tl/bake/development.rst]: custom to pasadya

File: tl/bake/development.rst [hunk 14]
```diff
@@ -267,15 +260,15 @@ add the following content::
...
+na pangalan ng command::
```
Comment (mind-blast) [inline tl/bake/development.rst]: command to utos

File: tl/bake/usage.rst [hunk 15]
```diff
@@ -1,19 +1,19 @@
...
 
-The cake console is run using the PHP CLI (command line interface).
-If you have problems running the script, ensure that:
+Ang cake na console ay pinapatakbo gamit ang PHP na CLI (command line interface).
+Kung may mga problema ka sa pagpapatakbo ng script, siguraduhin na:
 
-#. You have the PHP CLI installed and that it has the proper modules enabled (eg: MySQL, intl).
-#. Users also might have issues if the database host is 'localhost' and should try '127.0.0.1' instead, as localhost can cause issues with PHP CLI.
-#. Depending on how your computer is configured, you may have to set execute rights on the cake bash script to call it using ``bin/cake bake``.
+#. Mayroon kang naka-install na PHP CLI at mayroon itong wastong mga modyul na pinagana (eg: MySQL, intl).
```
Comment (mind-blast) [inline tl/bake/usage.rst]: eg to hal

File: tl/bake/usage.rst [hunk 16]
```diff
@@ -115,11 +115,11 @@ available using the ``--help`` option::
                        Bake|Migrations)
     --verbose, -v      Enable verbose output.
 
-Bake Themes
+Mga Tema ng Bake
 ===========
 
-The theme option is common to all bake commands, and allows changing the bake
-template files used when baking. To create your own templates, see the
+Ang opsyon ng tema ay pangkaraniwan sa lahat ng mga command ng bake, at nagpapahintulot sa pagbabago ng bake
```
Comment (mind-blast) [inline tl/bake/usage.rst]: command  to utos

File: tl/bake/usage.rst [hunk 17]
```diff
@@ -115,11 +115,11 @@ available using the ``--help`` option::
...
     --verbose, -v      Enable verbose output.
 
-Bake Themes
+Mga Tema ng Bake
 ===========
 
-The theme option is common to all bake commands, and allows changing the bake
-template files used when baking. To create your own templates, see the
+Ang opsyon ng tema ay pangkaraniwan sa lahat ng mga command ng bake, at nagpapahintulot sa pagbabago ng bake 
+na template na mga file na ginagamit sa pagbi-bake. Para lumikha ng iyong sariling mga templates, tingnan ang
```
Comment (mind-blast) [inline tl/bake/usage.rst]: templates to template

File: tl/console-and-shells/cache.rst [hunk 18]
```diff
@@ -1,8 +1,8 @@
-Cache Shell
+Shell ng Cache
 ===========
 
-To help you better manage cached data from a CLI environment, a shell command
-is available for clearing cached data your application has::
+Upang tulungan ka ng mas mahusay na pamamahala ng cached na data mula sa isang CLI na environment, isang command ng shell
```
Comment (mind-blast) [inline tl/console-and-shells/cache.rst]: command to utos

Comment (mind-blast) [reply tl/console-and-shells/cache.rst]: data  to datos

File: tl/console-and-shells/cache.rst [hunk 19]
```diff
@@ -1,8 +1,8 @@
-Cache Shell
+Shell ng Cache
 ===========
 
-To help you better manage cached data from a CLI environment, a shell command
-is available for clearing cached data your application has::
+Upang tulungan ka ng mas mahusay na pamamahala ng cached na data mula sa isang CLI na environment, isang command ng shell 
+ay magagamit sa pag-clear ng cached na data na meron ang iyong aplikasyon::
```
Comment (mind-blast) [inline tl/console-and-shells/cache.rst]: pag-clear to paglilinaw

**Comment by mind-blast** [comment]
Please check on my suggestions.

**Comment by smervs** [comment]
Done updating the files

File: tl/bake/development.rst [hunk 20]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
+#. Kapag nagpapatakbo ng bake gamitin ang ``--theme`` na opsyon upang tiyakin ang bake-theme 
+   na gusto mong gamitin. Upang maiwasan ang pagkakaroon ng pagtiyak ng opsyon na ito sa bawat tawag, maaari mo ring
+   itakda ang iyong pasadya na tema na gagamitin bilang isang default na tema::
 
         <?php
         // in config/bootstrap.php or config/bootstrap_cli.php
         Configure::write('Bake.theme', 'MyTheme');
 
-Customizing the Bake Templates
+Ang Pagcustomize ng mga Template ng Bake
```
Comment (severinolorillajr) [inline tl/bake/development.rst]: Pag-customize

File: tl/bake/development.rst [hunk 21]
```diff
@@ -165,66 +158,66 @@ And the resultant baked class (**src/Shell/FooShell.php**) looks like this::
...
+Kung nais mong baguhin ang default na output na ginawa ng utos ng "bake", maaari kang
+lumikha ng iyong sariling mga template ng bake sa iyong aplikasyon. Sa ganitong paraan ay hindi ginagamit ang 
+``--theme`` na opsyon sa command line habang nag-bake. Ang pinakamahusay na paraan upang gawin ito ay:
 
-#. Create a new directory **/src/Template/Bake/**.
-#. Copy any templates you want to override from
-   **vendor/cakephp/bake/src/Template/Bake/** to matching files in your
-   application.
+#. Maglikha ng bagong directory **/src/Template/Bake/**.
+#. Kopyahin ang anumang mga template ang gusto mong i-override mula sa
```
Comment (severinolorillajr) [inline tl/bake/development.rst]: >template na gusto

File: tl/bake/development.rst [hunk 22]
```diff
@@ -252,9 +245,9 @@ FooTask.php file should look like::
 
     }
 
-Once this file has been created, we need to create a template that bake can use
-when generating code. Create **src/Template/Bake/foo.twig**. In this file we'll
-add the following content::
+Sa sandaling nalikha ang file na ito, Kailangan nating lumikha ng isang template na maaaring gamitin ng bake
```
Comment (severinolorillajr) [inline tl/bake/development.rst]: >ito, kailangan nating maglikha

File: tl/bake/development.rst [hunk 23]
```diff
@@ -267,15 +260,15 @@ add the following content::
...
+Dapat mo na ngayong makita ang bago mong gawain sa output ng ``bin/cake bake``. Kaya mong 
+patakbuhin ang iyong bagong gawain sa pamamagitan ng pagpapatakbo ng ``bin/cake bake foo Example``.
+Ito ay bubuo ng isang bagong klase ng ``ExampleFoo`` sa **src/Foo/ExampleFoo.php**
+para gamitin sa iyong aplikasyon.
 
-If you want the ``bake`` call to also create a test file for your
-``ExampleFoo`` class, you need to overwrite the ``bakeTest()`` method in the
-``FooTask`` class to register the class suffix and namespace for your custom
-command name::
+Kung nais mo na ang ``bake`` na tawag ay maglikha din ng isang test na file para sa iyong
```
Comment (severinolorillajr) [inline tl/bake/development.rst]: >``bake`` na pagtawag

File: tl/console-and-shells/cache.rst [hunk 24]
```diff
@@ -1,8 +1,8 @@
-Cache Shell
+Shell ng Cache
 ===========
 
-To help you better manage cached data from a CLI environment, a shell command
-is available for clearing cached data your application has::
+Upang tulungan ka ng mas mahusay na pamamahala ng cached na data mula sa isang CLI na environment, isang utos ng shell
```
Comment (severinolorillajr) [inline tl/console-and-shells/cache.rst]: Para matulungan na mas mahusay na mapamahalaan ang

**Comment by smervs** [comment]
Done updating the files. Thank you for the proofread.

**Pull request merged**